src/base/decoder/types.ts

```diff
@@ -19,6 +19,10 @@ export function isArray (type: string): boolean {
   return type.includes('[]') && type.lastIndexOf(']') === type.length - 1;
 }
 
+export function isHex(data: string): boolean {
+  return /^([0-9a-f])+$/.test(data);
```

Comment:
does it make sense to handle '0x....' hex strings here? 
Maybe with something like `(?<=(0x)|^)([0-9a-f]+$)`

Reply:
I have to remove 0x anyway to perform a Buffer.from(data), to me it does not make sense to handle 0x in this validation.